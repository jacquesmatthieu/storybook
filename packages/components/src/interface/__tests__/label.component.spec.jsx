import React from 'react';
import { shallow } from 'enzyme';

import Label from '../label.component';

describe('Label', () => {
  let props;

  beforeEach(() => {
    props = {
      color: 'foo',
    };
  });

  const getWrapper = () => shallow(<Label {...props}>Bar</Label>);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
