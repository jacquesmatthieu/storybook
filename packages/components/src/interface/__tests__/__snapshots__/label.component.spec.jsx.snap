// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render properly 1`] = `
<labelcomponent__LabelWrapper
  color="foo"
  variant="input"
>
  Bar
</labelcomponent__LabelWrapper>
`;
